doc/whats_new.rst

```diff
@@ -67,6 +69,8 @@ API
 
 - Deprecated `postproc` argument in :func:`~hnn_core.dipole.simulate_dipole`, whereby user should explicitly smooth and scale resulting dipoles, by `Christopher Bailey`_ in `#372 <https://github.com/jonescompneurolab/hnn-core/pull/372>`_
 
+- Number of drive cells can now be specified through the `n_drive_cells` argument in `Network.add_xxx_drive()` methods, by `Ryan Thorpe`_ in `#383 <https://github.com/jonescompneurolab/hnn-core/pull/383>`_
```

Comment:
I think what is new should be moved above and what is being deprecated should be mentioned here